interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -424,12 +425,25 @@ MathFuncts, errGen) {
             MathFuncts.mean(nodeObject.activityA) -
             MathFuncts.mean(nodeObject.activityB)
           );
-          nodeObject.logsig = -Math.log10(MathFuncts.tTest(
+          nodeObject.rawPValue = MathFuncts.tTest(
             nodeObject.activityA, nodeObject.activityB
-          ).pValue());
+          ).pValue();
 
           return nodeObject;
         });
+
+        // use FDR on the raw p-values from nodeInfoSet to get adjustedPValues
+        var rawPValues = nodeInfoSet.map(function getRawPValue(nodeObject) {
+          return nodeObject.rawPValue;
+        });
+        var adjustedPValues = MathFuncts.multTest.fdr(rawPValues);
+
+        // map the adjustedPValues back into the nodeInfoSet
+        nodeInfoSet = nodeInfoSet.map(function(nodeObject, i) {
```

Comment:
Aha. A great question. I feel like I was avoiding `forEach()` for some reason... let me see.

By the way, there's nothing wrong with being opinionated! It's being inflexible *and* opinionated where the trouble starts. 😄